src/tab.group.ts

```diff
@@ -105,6 +101,15 @@ export default class TabGroup extends LitElement {
           >
             <!-- @type {Tab} -->
           </slot>
+
+          <div
```

Comment:
Changed this from a pseudo element to an actual element so I can measure its width in tests.